### PR TITLE
net-p2p/cardano-node: Update to 1.35.2 + fixes

### PR DESCRIPTION
**Pull request by reqlez**

### CODE REVIEW

**Comment by reqlez** [comment]
@arrowd seems 1.35.3 is not coming this weekend, so bumping for now to .2 + command fix.

**Comment by reqlez** [comment]
Forgot to mention, i'm adding cardano-tracer to build, we can use this one later for monitoring... i'm still in the process of testing that component and working out an rc file for it. It's in "MVP" state right now ( the tracer / RTView )

File: net-p2p/cardano-node/Makefile [hunk 1]
```diff
@@ -48,7 +48,7 @@ BINARY_ALIAS=		git=true
 USERS=		cardano
 GROUPS=		cardano
 
-USE_CABAL=	Diff-0.4.1 \
+USE_CABAL=      Diff-0.4.1 \
```
Comment (arrowd) [inline net-p2p/cardano-node/Makefile:51]: Is this a space?

**Review by arrowd — CHANGES_REQUESTED** [review]

File: net-p2p/cardano-node/Makefile (hunk 1 above)
Comment (reqlez) [reply net-p2p/cardano-node/Makefile:51]: Of course it is... how did it get there...

Comment (reqlez) [reply net-p2p/cardano-node/Makefile]: fixed

**Pull request merged**